packages/redirect-parser/package.json

```diff
@@ -15,7 +15,7 @@
     "test": "vitest run",
     "test:bench": "vitest bench",
     "test:dev": "vitest",
-    "test:ci": "vitest run --reporter=default && vitest bench --run"
+    "test:ci": "vitest run --reporter=default && vitest bench --run --passWithNoTests"
```

Comment:
In node 14 we won't run any bench tests and if tests=0 the default vitest behaviour is to exit with exit code 1